engine/config/1.0/schema.json

```diff
@@ -359,32 +298,30 @@
                         "$ref": "#/$defs/oauthRequestConfig"
                       },
                       "user_info_request": {
-                        "type": "object",
-                        "properties": {
-                          "endpoint": { "type": "string" },
-                          "method": {
-                            "type": "string",
-                            "enum": ["GET", "POST"]
-                          },
-                          "auth_method": { "type": "string" },
-                          "parameters": { "type": "object" },
-                          "request_content_type": { "type": "string" },
-                          "response_content_type": { "type": "string" },
-                          "response_map": {
-                            "type": "object",
-                            "additionalProperties": { "type": "string" }
+                        "allOf": [
+                          {
+                            "$ref": "#/$defs/oauthRequestConfig"
```

Comment:
Re-used `oauthRequestConfig` here like the other request properties, with the addition of `triggers`.